### ISSUE TITLE
Using a PageWrapper component to streamline page creation

### ISSUE THREAD

**Issue by meowwwls** [teacherfund/TeacherFund_next#151]
Though we only have a few pages now, I know it will grow. I was thinking it would help to have a `PageWrapper` component for all of the pages that will be the same except for the content portion. This will not only make it easier to create new pages, but will give us a more central location to change things across all pages, should they need to change in the future. 

For example, the pages really should have a skip link for accessibility, allowing people using assistive technology and the keyboard to skip easily to the main content and not have to tab through all of the navigation links. This would have to be added to every page, is easy to forget, and is easy to get wrong (like forgetting to set `tabindex="-1"` so it can be programmatically focused, but not by the keyboard, or adding the link in the wrong position in the markup). Including that in a `PageWrapper` component would remove chances of forgetting it or getting it wrong and people creating new pages would not even have to know about it.

It also allows us to do fewer imports in each page file, offloading most of that work to the `PageWrapper` component.

Our `PageWrapper` component could look like:

```jsx
import Nav from '../components/nav'
import Footer from '../components/footer'
import Head from '../components/head'

import '../static/styles/main.scss'

const PageWrapper = (props) => {
  return (
    <>
      <Head title={props.title} />
      <a href="#main-content" className='skip-link'>Skip to main content</a>
      <Nav />
      <main id="main-content" tabindex="-1">
        {props.children}
      </main>
      <Footer />
    </>
  )
}

export default PageWrapper
```

and then, for example, our Mission page would be simplified into:

```jsx
import PageWrapper from '../components/pageWrapper'
import { Component } from 'react' // only really needed if we need state, which for this page we won't  
import Link from 'next/link'

class Mission extends Component {
  return (
    <PageWrapper title="Our Mission – The Teacher Fund">
      // all of page content 
    </PageWrapper>
  )
}
```

This will help simplify the creation of new pages and also make sure skip links are always included and with the best practices being followed.

Any thoughts? If it's decided it's a good idea, I could work on it if no one minded.
